docs/content/server.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2024-03-14
+title: Grizzly Server
+---
+## An HTTP Server for editing and reviewing
+
+Do you have Grafana resources, dashboards, datasources, alerts, etc, on disk,
+and want a way to edit or review them before publishing?
+
+Grizzly Server is here to help.
```

Comment:
Feel free to ignore this if you don't agree 😄 : Personally, I like my docs drier and less telemarkety

Reply:
Will downplay. But that's bigtime what this PR does in general with the docs - they're way OTT at the moment.